website/docs/docs/cloud/about-cloud-develop.md

```diff
@@ -0,0 +1,20 @@
+---
+title: About developing in dbt Cloud
+id: about-cloud-develop
+description: "Learn how to develop your dbt projects using dbt Cloud."
+sidebar_label: "About developing in dbt Cloud"
+pagination_next: "docs/cloud/cloud-cli-installation"
+---
+
+dbt Cloud offers a fast and reliable way to work on your dbt project. You can use it through an integrated development environment (IDE) in your browser or a dbt Cloud-powered command line interface (CLI):
+
+- **[dbt Cloud CLI](/docs/cloud/cloud-cli-installation)** &mdash; Allows you to develop and run dbt commands from your local command line or code editor against your dbt Cloud development environment.
+
+  - Something to note is that the dbt Cloud CLI and the open-sourced dbt Core are both command line tools that let you run your dbt projects and use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure and integrates those builds with all [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
+
+- **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface (UI).
```

Comment:
Suggest not identifying "UI" unless you're going to use it later on the page or in the paragraph.
```suggestion
- **[dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud)** &mdash; Develop directly in your browser, making dbt project development efficient by compiling code into SQL and managing project changes seamlessly using an intuitive user interface.
```